libs/payments/ui/src/lib/client/components/CheckoutForm.tsx

```diff
@@ -0,0 +1,126 @@
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      // Stripe.js hasn't yet loaded.
+      // Make sure to disable form submission until Stripe.js has loaded.
+      return;
+    }
+
+    setLoading(true);
+
+    // Trigger form validation and wallet collection
+    const { error: submitError } = await elements.submit();
+    if (submitError) {
+      //await handleStripeErrorAction(cart.id, cart.version, submitError);
+      console.log(submitError);
+      return;
+    }
+
+    const result = await checkoutCartWithStripe(cart.id, cart.version, '');
+
+    const { type, clientSecret } = result as any;
+    const confirmIntent =
+      type === 'setup' ? stripe.confirmSetup : stripe.confirmPayment;
+
+    // Confirm the Intent using the details collected by the Payment Element
+    const { error } = await confirmIntent({
+      elements,
+      clientSecret,
+      confirmParams: {
+        return_url: successRedirectUrl,
+        payment_method_data: {
+          billing_details: {
+            address: {
+              country: cart.taxAddres?.countryCode,
+              postal_code: cart.taxAddres?.postalCode,
+            },
+          },
+        },
+      },
+    });
+
+    if (error) {
+      // This point is only reached if there's an immediate error when confirming the Intent.
+      // Show the error to your customer (for example, "payment details incomplete").
+      await handleStripeErrorAction(cart.id, cart.version, error);
+    } else {
+      // Your customer is redirected to your `return_url`. For some payment
+      // methods like iDEAL, your customer is redirected to an intermediate
+      // site first to authorize the payment, then redirected to the `return_url`.
+    }
+  };
+
+  return (
+    <form onSubmit={submitHandler} className="flex flex-col gap-4">
+      <label>
+        <span className="font-medium text-sm text-grey-400 block mb-2 text-start">
+          Name as it appears on your card
+        </span>
+        <input
+          type="text"
+          name="name"
+          className="w-full h-12 border rounded border-black/30 px-4"
+          data-testid="name"
+          placeholder="Full Name"
+          required
+          spellCheck={false}
+          disabled={false}
+        />
+      </label>
+      <PaymentElement
+        options={{
+          fields: {
+            billingDetails: {
+              address: {
+                postalCode: 'never',
+                country: 'never',
+              },
+            },
+          },
+        }}
+      />
+      <button
+        className="flex items-center justify-center bg-blue-500 font-semibold h-12 rounded-md text-white w-full p-4 mt-6 disabled:relative disabled:after:absolute disabled:after:content-[''] disabled:after:top-0 disabled:after:left-0 disabled:after:w-full disabled:after:h-full disabled:after:bg-white disabled:after:opacity-50 disabled:after:z-[100] disabled:border-none"
```

Comment:
There is a `z-50` on https://tailwindcss.com/docs/z-index. 

#### Checking for custom `z-[nnn]` values
<details>
<summary><kbd>git grep -IPn "\bz-\[\d+\]" | cat</kbd></summary>

```sh
apps/payments/next/app/styles/global.css:62:  @apply payment-button-disabled cursor-not-allowed after:bg-white after:opacity-50 after:z-[5];
libs/payments/ui/src/lib/client/components/CheckoutForm.tsx:118:        className="flex items-center justify-center bg-blue-500 font-semibold h-12 rounded-md text-white w-full p-4 mt-6 disabled:relative disabled:after:absolute disabled:after:content-[''] disabled:after:top-0 disabled:after:left-0 disabled:after:w-full disabled:after:h-full disabled:after:bg-white disabled:after:opacity-50 disabled:after:z-[100] disabled:border-none"
packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx:14:      className="flex justify-center items-center fixed inset-0 z-[9999]"
packages/fxa-payments-server/src/components/PayPalButton/index.tsx:250:    ' payment-button-disabled after:bg-white after:opacity-40 after:z-[100]';
packages/fxa-payments-server/src/components/PaymentForm/index.tsx:271:    'payment-button-disabled after:bg-white after:opacity-50 after:z-[100] border-none';
packages/fxa-payments-server/src/components/fields/index.tsx:452:        ' payment-button-disabled cursor-not-allowed after:bg-white after:opacity-50 z-[5] border-none focus:border-solid focus:border-2 focus:border-offset-2 focus:border-blue-400'
packages/fxa-payments-server/src/styles/tailwind.css:154:  @apply payment-button-disabled cursor-not-allowed after:bg-white after:opacity-50 after:z-[5];
packages/fxa-payments-server/src/styles/tailwind.css:160:  @apply z-[6] px-4 py-1.5 left-0 -mt-12;
packages/fxa-react/components/Portal/index.tsx:44:        'portal fixed top-0 inset-x-0 p-0 z-[9999] tablet:px-16'
packages/fxa-settings/src/components/Settings/HeaderLockup/index.tsx:28:        className="desktop:hidden me-6 p-2 self-center -m-2 z-[1] rounded hover:bg-grey-100"
packages/fxa-settings/src/components/Settings/HeaderLockup/index.tsx:70:        className="inline-block relative p-2 -m-2 z-[1] rounded hover:bg-grey-100"
```
</details>

---

#### Checking for Tailwind `z-nnn` values
<details>
<summary><kbd>git grep -IPn "\bz-\d+" -- ":(exclude)*.txt" | cat</kbd></summary>

```sh
apps/payments/next/app/layout.tsx:25:          className="bg-white fixed flex justify-between items-center shadow h-16 left-0 top-0 mx-auto my-0 px-4 py-0 w-full z-10 tablet:h-20"
libs/shared/assets/src/styles/tooltips.css:6:  @apply absolute text-start w-[calc(100%+.5rem)] -left-1 desktop:w-64 desktop:left-auto px-5 py-4 bg-white rounded-sm top-[calc(100%+.25rem)] desktop:-top-2 border border-grey-100 shadow-sm transition-opacity z-10;
packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:105:        className="p-2 border-b border-grey-100 block hover:bg-grey-10 focus:bg-grey-10 z-50"
packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:172:              className="suggestions-list absolute top-full w-full bg-white border border-grey-100 mt-3 shadow-sm rounded overflow-hidden z-50"
packages/fxa-payments-server/src/components/Header/index.tsx:30:      className={`${className} flex justify-between items-center bg-white shadow h-16 fixed left-0 top-0 my-0 mx-auto px-4 py-0 w-full z-10 tablet:h-20`}
packages/fxa-payments-server/src/components/PayPalButton/index.tsx:259:          className={`paypal-button w-60 m-auto mt-14 mb-16 relative z-0${
packages/fxa-react/components/Header/index.tsx:19:      className={props.className || 'sticky top-0 bg-grey-10 z-10'}
packages/fxa-react/styles/tooltips.css:6:  @apply absolute text-start w-[calc(100%+.5rem)] -left-1 desktop:w-64 desktop:left-auto px-5 py-4 bg-white rounded-sm top-[calc(100%+.25rem)] desktop:-top-2 border border-grey-100 shadow-sm transition-opacity z-10;
packages/fxa-settings/.storybook/design-guide/Copiable.tsx:19:      <button className="copy-content hidden absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 z-10 text-sm pointer bg-white text-grey-900 focus:bg-grey-100 px-1 shadow-md rounded border border-grey-200 rounded-md whitespace-nowrap font-mono pr-7">
packages/fxa-settings/.storybook/design-guide/Header.tsx:6:    <div className="bg-grey-50 flex items-center justify-between sticky z-20 top-0 py-4 pb-3 border-b border-grey-400/25 px-2 -mt-4 -ml-12 -mr-12">
packages/fxa-settings/src/components/Settings/AlertBar/index.tsx:46:      className="fixed z-10"
packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx:54:          className={`w-full h-full fixed top-0 start-0 bg-white z-10
packages/fxa-settings/src/components/Settings/ContentSkip/index.tsx:11:    className="sr-only focus:w-auto focus:h-auto focus:clip-auto absolute bg-white font-medium shadow-md px-3 py-2 z-10 rounded ml-2 mt-2 cursor-pointer"
packages/fxa-settings/src/components/Settings/Modal/index.tsx:61:        className="flex flex-col justify-center fixed inset-0 z-50 w-full px-2 h-full bg-black/75"
packages/fxa-settings/src/components/Tooltip/index.tsx:52:        `z-50 absolute py-2 px-6 text-center text-white
```
</details>

Reply:
I'll replace with z-20.